src/backend/InvenTree/InvenTree/settings.py

```diff
@@ -1298,9 +1298,7 @@
 
 HEADLESS_ONLY = True
 HEADLESS_TOKEN_STRATEGY = 'InvenTree.auth_overrides.DRFTokenStrategy'
-MFA_ENABLED = get_boolean_setting(
```

Comment:
That is still relevant

Reply:
Ok, sorry about that! Thought you had addressed this in the recent merge.

Reply:
I did but there are no satisfactory tests